hw/dv/tools/dvsim/testplans/sec_cm_count_testplan.hjson

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: hardened_counter_check
```

Comment:
Will be nice to have the name of the prim module reflected in the name. Also, I thought we were following some conventions in the vseq too right? We should match everything up.

Reply:
sure, updated to prim_count_check. vseq uses prim_count.